tests/Jackalope/Transport/DoctrineDBAL/XmlPropsRemover/XmlPropsRemoverTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Jackalope\Transport\DoctrineDBAL\XmlPropsRemover;
+
+use Jackalope\Factory;
+use Jackalope\Test\TestCase;
+use Jackalope\Transport\DoctrineDBAL\XmlParser\XmlToPropsParser;
+use PHPCR\Util\ValueConverter;
+
+class XmlPropsRemoverTest extends TestCase
```

Comment:
can you please add a test that requires escaping of name/value and makes sure that all works as expected too?

Reply:
@dbu yeah I will check the JCR specification for what is even possible as jcr property names. The jcr values are already escaped tested via `ampersand` property.

Reply:
Additional tests are added for more xml special chars `<>&'"` and tested